hack/make/.build-deb/rules

```diff
@@ -6,7 +6,7 @@ override_dh_gencontrol:
 	# if we're on Ubuntu, we need to Recommends: apparmor
 	echo 'apparmor:Recommends=$(shell dpkg-vendor --is Ubuntu && echo apparmor)' >> debian/docker-engine.substvars
 	# if we are building experimental we reccomend yubico-piv-tool
-	echo 'yubico:Recommends=$(shell [ "$DOCKER_EXPERIMENTAL" ] && echo "yubico-piv-tool (>= 1.1.0~)")' >> debian/docker-engine.substvars
+	echo 'yubico:Recommends=$(shell [ "$DOCKER_EXPERIMENTAL" ] && echo "yubico-piv-tool (>= 1.1.0~), libykcs11-1 (>= 1.1.0~)")' >> debian/docker-engine.substvars
```

Comment:
you da bomb i will close for time being :)

On Thu, Jan 7, 2016 at 3:23 PM, Tianon Gravi <notifications@github.com>
wrote:

> In hack/make/.build-deb/rules
> <https://github.com/docker/docker/pull/19168#discussion_r49140506>:
>
> > @@ -6,7 +6,7 @@ override_dh_gencontrol:
> >  	# if we're on Ubuntu, we need to Recommends: apparmor
> >  	echo 'apparmor:Recommends=$(shell dpkg-vendor --is Ubuntu && echo apparmor)' >> debian/docker-engine.substvars
> >  	# if we are building experimental we reccomend yubico-piv-tool
> > -	echo 'yubico:Recommends=$(shell [ "$DOCKER_EXPERIMENTAL" ] && echo "yubico-piv-tool (>= 1.1.0~)")' >> debian/docker-engine.substvars
> > +	echo 'yubico:Recommends=$(shell [ "$DOCKER_EXPERIMENTAL" ] && echo "yubico-piv-tool (>= 1.1.0~), libykcs11-1 (>= 1.1.0~)")' >> debian/docker-engine.substvars
>
> https://bugs.debian.org/810284
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/19168/files#r49140506>.
>